src/app/tests/TestCommandInteraction.cpp

```diff
@@ -253,10 +253,12 @@ class TestCommandInteraction
     static void TestCommandHandlerWithSendEmptyResponse(nlTestSuite * apSuite, void * apContext);
 
     static void TestCommandHandlerWithProcessReceivedEmptyDataMsg(nlTestSuite * apSuite, void * apContext);
+#if CHIP_CONFIG_SENDING_BATCH_COMMANDS_ENABLED
```

Comment:
Question really should be, do we want to allow all device to support sending out batch commands if they wanted to?

Otherwise this is needed because NRF does compile and run these unit tests so they fail to build and the CommandSender is not enabled to send out batch commands